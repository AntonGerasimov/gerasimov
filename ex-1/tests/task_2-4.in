2147483637 1234123
