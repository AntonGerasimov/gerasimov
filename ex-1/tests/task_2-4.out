2147483647 + 1234123
